pkgimage.mk

```diff
@@ -24,129 +24,5 @@ $(JULIA_DEPOT_PATH):
 print-depot-path:
 	@$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --startup-file=no -e '@show Base.DEPOT_PATH')
 
-all-release: $(addprefix cache-release-, $(STDLIBS))
-all-debug:   $(addprefix cache-debug-, $(STDLIBS))
-
-define stdlib_builder
-ifneq ($(filter $(1),$(INDEPENDENT_STDLIBS) $(PKG_EXTS)),)
-# Define target-specific export for `JULIA_CPU_TARGET`
```

Comment:
Is `JULIA_CPU_TARGET` handled currently?

Reply:
How should it be?